src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is there any reason for adding this config?

Reply:
The XML declaration is optional in version 1.0 and it would work without it as well. But the spec says it "SHOULD" be there and also in 1.1 it is mandatory, so personally I just add it to every XML document to avoid potential problems by the parser guessing the wrong encoding or so. See https://stackoverflow.com/questions/7007427/does-a-valid-xml-file-require-an-xml-declaration for more details.